### DISCUSSION TITLE
Welcome !

### DISCUSSION THREAD

**Discussion by zong-zhe** [KusionStack/kusion#155] [Ideas]
1. 日常工作中有参与过哪些 K8s 运维相关的工作？遇到了哪些困难？
2. 使用 GoCity 后的直观感受，Platform as Code / Infrastructure-as-code 与您目前接触过的运维场景有什么不同。
3. 如果让您考虑采用 KusionStack 解决日常工作中的一些问题，您可能会从哪几个方面调研 KusionStack 是否满足您的需要。
4. 你期待的应用交付/运维理想画面是怎样的？
5. 在你工作中，有接触或了解过 DevOps 么，理想中的 DevOps 工具是什么样的？
6. 对 KusionStack 或 Devjoy 的建议。

**Comment by 061501** [comment]
今天参展商的小姐姐都好漂亮哈哈哈😁

**Comment by vvheartattack** [comment]
5. not yet

**Comment by airmelt** [comment]
6. 下载太慢了，配置过程比较复杂

**Reply by He1pa** [reply]
hello，请问您是当天带电脑来体验的同学么？我记得当时是安装minikube出现了一些问题，方便提供一下报错信息么

**Comment by Edward088** [comment]
不大了解，后续可以关注一下

**Comment by panzd** [comment]
之前都是用的terraform和ansible，日常开发还是有点麻烦的，kusion使用起来更加便捷，k文件比yaml更加简单，搭建服务似乎更快，与k8s交互也很智能，希望kusion越做越好。

**Comment by JohnLuCN** [comment]
回家跑一下命令行体验体验～

**Comment by lixincao1999** [comment]
终于登上GitHub了😭

**Comment by imyuanx** [comment]
希望可以减少配置，快速部署

**Comment by He1pa** [comment]
1. 端口转发可不可以写到 yaml里，不需要 kubectl port-forward svc/gocity 4000:4000 -n code-city
2. 能不能部署到不同的云
3. 除了github，能不能在其他平台搭建
4. gocity的案例操作内容有点少，只有发布
5. 跟Terraform、HCL/ AWS CDK的区别
6. 有没有做UI，只能通过命令行的方式执行么？
7. 仓库里有openkruise，prometheus的内容，跟它们的关系是怎样的？
8. 目前有盈利么？打算怎么盈利

**Reply by He1pa** [reply]
Q: 6 有没有做UI，只能通过命令行的方式执行么？
A: 基于 IaC （Infrastructure-as-Code）的思路，配置的定义和管理还是通过 code 的方式来实现，而不是操作用户界面的形式。但在 kusion 的一些能力上，例如apply，watch，将来会集成在 IDE 中。

**Comment by airmelt** [comment]
Can't find/load kube config file. It seems it should be in my Macpro(M1)
user root folder.

HeiPa ***@***.***> 于2022年11月7日周一 15:46写道：
